packages/api/prisma/schema.prisma

```diff
@@ -14,29 +14,54 @@ datasource db {
 }
 
 model Book {
-  id      Int       @id
-  name    String
-  search  String    @unique // This gives us a human readable key to look up books from URL params.
-  Chapter Chapter[]
-}
-
-model Chapter {
   id     Int     @id
-  number Int
-  book   Book    @relation(fields: [bookId], references: [id])
-  bookId Int
+  name   String
+  search String  @unique // This gives us a human readable key to look up books from URL params.
   Verse  Verse[]
-
-  @@unique([bookId, number])
 }
 
 model Verse {
-  id        Int     @id
-  number    Int
-  chapter   Chapter @relation(fields: [chapterId], references: [id])
-  chapterId Int
+  id      Int    @id
+  number  Int
+  chapter Int
+  book    Book   @relation(fields: [bookId], references: [id])
+  bookId  Int
+  Word    Word[]
+
+  @@unique([chapter, number])
+}
+
+model Word {
+  id      String    @id
+  text    String
+  verse   Verse     @relation(fields: [verseId], references: [id])
+  verseId Int
+  order   Int
+  form    LemmaForm @relation(fields: [formId], references: [id])
+  formId  String
```

Comment:
A word is connected to a form, which has the grammar information and connects the word to its lemma (dictionary entry)